youtube_dl/extractor/sbs.py

```diff
@@ -36,8 +52,21 @@ class SBSIE(InfoExtractor):
         'only_matching': True,
     }]
 
+    def video_id_from_page_contents(self, url):
+        page_contents = self._download_webpage(url, None)
```

Comment:
No such id `None`.

Reply:
`id` doesn't seem to be used in this flow.